src/components/text-editor/prosemirror-adapter/menu/types.ts

```diff
@@ -15,6 +15,10 @@ export const EditorMenuTypes = {
     Link: 'link',
     OrderedList: 'ordered_list',
     BulletList: 'bullet_list',
+    Underline: 'underline',
+    Strikethrough: 'strikethrough',
+    Code: 'code',
+    CodeBlock: 'code_block',
```

Comment:
This PR is broken into commits in quite a weird way, imo. I'd rather see that any changes that were already needed before adding new types here were made first, in one or more commits. (I'm guessing one commit, since it's basically fixing one problem: "be able to toggle stuff on and off".) Then any new features, like support for underline and strikethrough, should be added in their own commits.

But, unless underline and strikethrough are considered MVP, we shouldn't add them now.

Reply:
I can re-arrange the commits in an order that makes more sense.